.travis.yml

```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: node_js
+cache:
+  npm: false
```

Comment:
O.o?

Reply:
yep, that seems to have fixed the testing on travis

Reply:
I guess it was my original over eagerness with updating all those modules :P

Reply:
Just fails on node 4 now

Reply:
huh, remove that and now it has cached again, grr